src/index.ts

```diff
@@ -114,6 +113,14 @@ export function apply(ctx: Context, config: Config) {
     .action(async ({ session, options }, input) => {
       if (!input?.trim()) return session.execute('help novelai')
 
+      // Check if the user is allowed to use this command.
+      // This code is originally written in the `resolution` function,
+      // but currently `session.user` is not available in the type infering process.
+      // See: https://github.com/koishijs/novelai-bot/issues/159
+      if (options.resolution?.custom && restricted(session)) {
+        return session.text('.custom-resolution-unsupported')
+      }
```

Comment:
Don't know why but `options.resolution` here is `any` rather than `Size`, is it bug or just my local configurations broke again?